#! /bin/sh /usr/share/dpatch/dpatch-run
## 16_fix-failed-SolrInfoMBeanTest.dpatch by Jan-Pascal van Best <janpascal@vanbest.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: See upstream issue SOLR-1253

@DPATCH@
diff -urNad solr~/src/test/org/apache/solr/SolrInfoMBeanTest.java solr/src/test/org/apache/solr/SolrInfoMBeanTest.java
--- solr~/src/test/org/apache/solr/SolrInfoMBeanTest.java	2009-07-02 09:28:07.000000000 +0200
+++ solr/src/test/org/apache/solr/SolrInfoMBeanTest.java	2009-07-02 09:28:09.000000000 +0200
@@ -81,7 +81,7 @@
     String path = pckgname.replace('.', '/');
     Enumeration<URL> resources = cld.getResources(path);
     while (resources.hasMoreElements()) {
-      directories.add(new File(URLDecoder.decode(resources.nextElement().getPath(), "UTF-8")));
+      directories.add(new File(resources.nextElement().getPath()));
     }
       
     ArrayList<Class> classes = new ArrayList<Class>();
